app/main/game.tsx

```
@@ -54,6 +105,244 @@ export function usePlayerColor(): 'w' | 'b' | null {
   return null;
 }
 
+const router = useRouter();
+// ✅ New component for the loading/waiting screen
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)